docs/content/en/docs/contribution-guidelines/control-plane-development.md

```diff
@@ -20,7 +20,7 @@ description: >
 Prepare a ControlPlane configuration file as described at [Installation](https://pipecd.dev/docs/operator-manual/control-plane/installation/) and start running by the following command:
 
 ``` console
-bazelisk run //cmd/server:server -- server \
+bazelisk run //cmd/pipecd -- server \
```

Comment:
I omitted it, but should I write it properly like `pipecd:pipecd`?

Reply:
yes, I think we should 🙏

Reply:
I updated 😄

Reply:
thanks 🙏